homeassistant/components/vacuum/__init__.py

```diff
@@ -68,6 +68,11 @@
 STATE_PAUSED = STATE_PAUSED
 STATE_RETURNING = 'returning'
 STATE_ERROR = 'error'
+STATE_REMOTE = 'remote controlled'
```

Comment:
Changes to the entity model need to be discussed and cleared in an issue in our architecture repo.

https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model

This is the current one about vaccum:
https://github.com/home-assistant/architecture/issues/29

But that one is more or less ready already.